sdk/swagger.yaml

```diff
@@ -290,6 +290,57 @@ components:
             $ref:  "#/components/schemas/OrgEnvironment"
         nextToken: 
           type: string
+    EnvironmentRevision:
+      type: object
+      properties:
+        number:
+          type: integer
+        creatorLogin:
+          type: string
+        created:
+          type: string
+        creatorName:
+          type: string
+        tags:
+          type: array
+          items:
+            type: string
+      required:
+        - number
+    EnvironmentRevisionTag:
+      type: object
+      properties:
+        revision:
+          type: integer
+        name:
+          type: string
+        created:
+          type: string
+        modified:
+          type: string
+        editorLogin:
+          type: string
+        editorName:
+          type: string
+      required:
+        - revision
+        - name

```

Comment:
I think `created` and `modified` are always present

Reply:
If we add this, the codegen will add those fields to the New like this:

func NewEnvironmentRevisionTag(revision int32, name string, created string, modified string) *EnvironmentRevisionTag

Because of the shape of the API, users really won't be using these methods, but in general we should think about the required fields as what the user has to supply to be valid, and leave the inferred fields out